torch/quantization/quantize_fx.py

```diff
@@ -20,13 +20,40 @@ def fuse_fx(graph_module, inplace=False):
     fuser = Fuser()
     return fuser.fuse(graph_module, inplace)
 
-def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic_quant):
+def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic_quant, is_standalone_module=False):
+    r""" Internal helper function for prepare_fx
+    Args:
+      graph_module, qconfig_dict, inplace: see docs for :func:`~torch.quantization.prepare_fx`
+      `is_standalone_module`: a boolean flag indicates whether we are
+      quantizing a standalone module or not, the way we quantize standalone module
+      is described in :func:`~torch.quantization.prepare_standalone_module_fx`
+    """
     _check_is_graph_module(graph_module)
     graph_module = fuse_fx(graph_module, inplace)
     quantizer = Quantizer()
     prepare = quantizer.prepare_dynamic if is_dynamic_quant else quantizer.prepare
-    prepared = prepare(graph_module, qconfig_dict, inplace=True)
-    return prepared
+    return prepare(graph_module, qconfig_dict, inplace=True, is_standalone_module=is_standalone_module)
+
+def _prepare_standalone_module_fx(graph_module, qconfig_dict, inplace=False):
+    r""" [Internal use only] Prepare a standalone module, so that it can be used when quantizing the
+    parent module. Used in custom module support
+
+    input of the module is quantized in parent module, output of the module
+    is quantized in the standalone module.
+    Returns:
+      model(GraphModule): prepared standalone module with following attributes:
+        _observed_input_idxs(List[Int]): a list of indexs for the graph inputs that
+                                         needs to be observed in parent module
+        _output_is_observed(Bool): a boolean variable indicate whether the output of the
+                                   custom module is observed or not
```

Comment:
would it be possible to also explain this in the file where the logic is?  I think duplicating the doc is worth not having to jump between different files.